File: domain/aplicativo/assinatura.py
class Assinatura:
    
    def __init__(self, usuario, plano):
        self.usuario = usuario
        self.plano = plano
        self.assinatura_ativa = True
    
    def criar_assinatura(cls, usuario, plano):
        assinatura = Assinatura(usuario, plano)
        plano.incluir_assinatura(assinatura)
        return assinatura

File: microsservicos/MusicApp.Usuario/MusicApp.Usuario.Domain/Conta/Aggregates/Assinatura.cs
using MusicApp.Usuario.Domain.Aggregates;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MusicApp.Usuario.Domain.Aggregates
{
    public class Assinatura
    {
        public Guid Id { get; set; }
        public Plano Plano { get; set; }
        public bool AssinaturaAtiva { get; set; }


        public Assinatura() {
            this.Id = Guid.NewGuid();
        }
    }


}
